python/src/robyn/modeling/pareto/pareto_optimizer.py

```diff
@@ -22,34 +23,6 @@
 from tqdm import tqdm  # Import tqdm for progress bar  # Import tqdm for progress bar
 
 
-@dataclass
-class ParetoResult:
-    """
-    Holds the results of Pareto optimization for marketing mix models.
-
-    Attributes:
-        pareto_solutions (List[str]): List of solution IDs that are Pareto-optimal.
-        pareto_fronts (int): Number of Pareto fronts considered in the optimization.
-        result_hyp_param (pd.DataFrame): Hyperparameters of Pareto-optimal solutions.
-        x_decomp_agg (pd.DataFrame): Aggregated decomposition results for Pareto-optimal solutions.
-        result_calibration (Optional[pd.DataFrame]): Calibration results, if calibration was performed.
-        media_vec_collect (pd.DataFrame): Collected media vectors for all Pareto-optimal solutions.
-        x_decomp_vec_collect (pd.DataFrame): Collected decomposition vectors for all Pareto-optimal solutions.
-        plot_data_collect (Dict[str, pd.DataFrame]): Data for various plots, keyed by plot type.
-        df_caov_pct_all (pd.DataFrame): Carryover percentage data for all channels and Pareto-optimal solutions.
-    """
-
-    pareto_solutions: List[str]
-    pareto_fronts: int
-    result_hyp_param: pd.DataFrame
-    x_decomp_agg: pd.DataFrame
-    result_calibration: Optional[pd.DataFrame]
-    media_vec_collect: pd.DataFrame
-    x_decomp_vec_collect: pd.DataFrame
-    plot_data_collect: Dict[str, pd.DataFrame]
-    df_caov_pct_all: pd.DataFrame
-
-
 @dataclass
 class ParetoData:
```

Comment:
Can this dataclass also moved to entities?

Reply:
It is only used as an object to return data from within pareto_optimizer. Its not really meant to be used outside of pareto_optimizer.